app/models/concerns/orchestration/tftp.rb

```diff
@@ -209,4 +208,9 @@ def each_unique_feasible_tftp_proxy
     end
     results.all?
   end
+
+  def local_boot_template_name(kind)
+    key = "local_boot_#{kind}"
```

Comment:
Nitpick: I'd prefer `kind.downcase` here but it's not exposed to the user not big deal.